README.md

```diff
@@ -36,13 +36,14 @@ Liota supports plain old BSD sockets, WebSocket and MQTT communication protocols
 * Identity class encapsulates certificates or credentials related to a connection used at both DCC and Device side.
 * TLSConf class encapsulates parameters related to TLS configuration.
 * CRLs(Certificate Revocation Lists) check has been implemented: [VRIFY_CRL_CHECK_CHAIN](https://docs.python.org/2/library/ssl.html#ssl.VERIFY_CRL_CHECK_CHAIN) in this mode CRLs of all certificates in the peer cert chain are checked.
- In order to enable the CRLs set the 'crl_path' in '/etc/liota/liota.conf', it requires a valid path to CRL in PEM format (preferable directory '/usr/lib/liota'). If the check is required to be disabled 'crl_path' should be left blank.
+In order to enable the CRLs set the 'crl_path' in '/etc/liota/liota.conf', it requires a valid path to CRL; if in case intermediate CA is used the CRL should include both Intermediate CRL and Root CRL in PEM format (preferable directory '/usr/lib/liota'). If the check is required to be disabled 'crl_path' should be left blank.
```

Comment:
In order to enable the CRLs set the 'crl_path' in '/etc/liota/liota.conf' --> In order to enable the CRLs set the 'crl_path' in liota.conf which by default is present in /etc/liota/.

Reply:
I believe the current statement which is in the code review provides a better understanding.

Reply:
The current statement makes it sound mandatory to modify '/etc/liota/liota.conf' even if the actual location of liota.conf is different. My statement was just a suggestion. Its ok if you dont want to change it.

Reply:
I plan to keep it in the current format.